### DISCUSSION TITLE
Is there a way to combine two different .ckpt files or safetensor files into one, without loosing any data?

### DISCUSSION THREAD

**Discussion by runner22k** [diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI#8] [Ideas]
Hi. I don't know anything about how AI or ML works. I saw some formulas which were used to merge ckpt files together. But none of those, from Automatic1111, can give me the result I wanted. So, I wanted to ask here, as you are able to work with the ckpt data. Is there a way to combine two different .ckpt files or safetensor files into one, without loosing any data? Can you make a tool or is there a tool already?

I want something like
C= A + (B-A)
while creating a new C.ckpt file by subtracting A from B before adding to A, so that we may reduce file size.

**Comment by diStyApps** [comment]
Hey ,
not sure of it will answer your specific requirements but you can try other GUI tool of mine:
[Merge-Stable-Diffusion-models-without-distortion-gui](https://github.com/diStyApps/Merge-Stable-Diffusion-models-without-distortion-gui)

Note that i didn't wrote the original code, i only wrote the GUI, did some basic tests that's it, so you can try if you can get better results.

Good luck.

**Reply by runner22k** [reply]
Thanks for the quick reply. Is that GUI windows based or Linux based?

**Comment by diStyApps** [comment]
For both, you got no install version for windows [https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases/download/0.1.1/Safe-and-Stable-Ckpt2Safetensors-GUI.v0.1.1.7z](https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases/download/0.1.1/Safe-and-Stable-Ckpt2Safetensors-GUI.v0.1.1.7z)
